### PR TITLE
add Ahihi

### PR DESCRIPTION
**Pull request by thaihungabh**
Ahihi Review Đi

### CODE REVIEW

**Comment by HuuHoc254** [comment]
hahahaha

File: clone/src/main/java/com/shopee/clone/config/TestMergeCode.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.shopee.clone.config;
+
+public class TestMergeCode {
```
Comment (minheagle) [inline clone/src/main/java/com/shopee/clone/config/TestMergeCode.java:3]: test comment

File: clone/src/main/java/com/shopee/clone/config/TestMergeCode.java
Comment (minheagle) [inline clone/src/main/java/com/shopee/clone/config/TestMergeCode.java:1]: test comment

Comment (HuuHoc254) [inline clone/src/main/java/com/shopee/clone/config/TestMergeCode.java:1]: đã review

Comment (thaihungabh) [inline clone/src/main/java/com/shopee/clone/config/TestMergeCode.java:1]: Let Review

File: clone/src/main/java/com/shopee/clone/config/TestMergeCode.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.shopee.clone.config;
```
Comment (minheagle) [inline clone/src/main/java/com/shopee/clone/config/TestMergeCode.java:1]: test comment 3

**Pull request merged**